ksqldb-functional-tests/src/test/resources/rest-query-validation-tests/pull-queries-against-materialized-aggregates.json

```diff
@@ -841,7 +841,7 @@
         {"admin": {"@type": "currentStatus"}},
         {"query": [
            {"header":{"schema":"`ID` STRING KEY, `WINDOWSTART` BIGINT KEY, `WINDOWEND` BIGINT KEY, `COUNT` BIGINT"}},
-          {"row":{"columns":["10", 1582501512000, 1582501513000, 1]}}
+          {"row":{"columns":["10", 1582501512000, 1582501513000, 2]}}
```

Comment:
1) How was this test passing all this time? 
2) Do we treat records with the same key+timestamp combination as a single record or multiple records while materializing aggregates?

Reply:
I think it passed before because it would consistently match against partial results.  It may also have been flaky.

For a windowed table, the table key is effectively the key+timestamp, so in this case, we get count 2 because the inputs share both a key and the same timestamp.  If we changed the timestamp of one of them to be outside the window, we would have two outputs, each with count 1.

Reply:
Hmm, I agree it was probably passing on an intermediate result. But how in the world did we write this expectation to begin with? It seems like it was obviously wrong. Oh, well, I agree it looks like it's right now.